src/Web/Views/Basket/CheckoutDetails.cshtml

```diff
@@ -168,6 +168,12 @@
             </div>
             <div class="row">
                 <div class="esh-catalog-items row">
+                    <div class="esh-basket-item">
```

Comment:
Move that to line 168:
 <section>
                    <div class="col-md-11">              
                        <div class="col-md-12">
                             <label for="environmentId">EnvironmentId</label>
                             @Html.TextBoxFor(model => model.EnvironmentId, new { @class = "form-control" })             
                        </div>
                    </div>
                </section>

It will look like that:
![image](https://user-images.githubusercontent.com/28061748/141051892-4429c9cf-9336-42b5-a5a3-fe549adb205c.png)

Reply:
Can you also shrink it so it's narrower like the other fields?  Also, label it "Environment ID" please.